lib/ui/contact_list/widgets/profile_ready_card.dart

```diff
@@ -93,7 +93,11 @@ class ProfileReadyCard extends ConsumerWidget {
           // Search For Friends button
           WnFilledButton(
             label: 'Search For Friends',
-            onPressed: () => NewChatBottomSheet.show(context),
+            onPressed: () {
+              // Dismiss the card when user takes action to search for friends
+              ref.read(profileReadyCardVisibilityProvider.notifier).dismissCard();
```

Comment:
But, if it doesn't start a chat, just searches, then I understand that we do want to show it right? Or not?

Reply:
@vladimir-krstic  WDYT?

Reply:
@untreu2  so maybe the change could be: first check if the card is visible and there are any groups. If so, call dismissCard(), instead of triggering it on the “start chat” button click.